account_chart_update/tests/test_account_chart_update.py

```diff
@@ -543,6 +543,14 @@ def test_00_matching(self):
         self.assertEqual(self.tax.description, self.tax_template.description)
         self.assertEqual(self.account.code, self.account_template.code)
         self.assertEqual(self.fp.name, self.fp_template.name)
+        fp_id = wizard.find_fp_by_templates(self.fp_template)
+        fp_rec = self.env["account.fiscal.position"].browse(fp_id)
+        expected_xmlid = "{}.{}_{}".format(
+            self.test_module,
+            wizard.company_id.id,
+            self.fp_template.get_external_id().get(self.fp_template.id).split(".")[1],
```

Comment:
Please put a fixed value here.

Reply:
This value cannot be fixed because it is generated based on the ID, which changes every time the test is run, as can be seen in:
https://github.com/OCA/account-financial-tools/blob/16.0/account_chart_update/tests/test_account_chart_update.py#L16

What I could do is format the first part of the xmlid (which is the name of the table and will be fixed) and format only the record ID instead of having to make all the calls and methods that I do here.
```python
"account_fiscal_position_template-{}".format(self.fp_template.id),
```

Do you think it would be better if I make the change I mentioned?

Reply:
I have made the change because I believe it makes the test more self-contained without relying on external implementations, thank you!